Flesh out README

J-Vaughan opened the pull request:
🚧 It's not ready yet, don't merge 🚧 
So all that's really left (for the first barebones readme) is:

- [ ] getting started guide react (#29)
- [ ] getting started guide express+nodejs (#30)
- [ ] Building and running from source (#30) <- Kinda like the stepsister of above; like, a more specialized guide on how to temporarily set up this project locally (maybe #29 ?), vs #30 being a more permanent 
- [x] Licensing (#32)
- [x] Write Coding guidelines (#33)
   - [x] a styleguide
   - [x] commit instructions

I will accept this in new PR Project roadmap (#31)

J-Vaughan committed:
Flesh out README

J-Vaughan committed:
docs: connected most links

There's still a couple links to non-existent files still

J-Vaughan commented:
So all that's really left (for the first barebones readme) is:

- [ ] getting started guide (me + someone who knows react)
- [ ] licensing (everyone + probably dave)
- [ ] a styleguide (saihaj/aidan?)

saihaj commented on README.md:
```diff
@@ -1,3 +1,28 @@
+[template](/.github/ISSUE_TEMPLATE/bug_report.md).
+
+If you're interested in contributing features or code, please review the
+[feature request template](/.github/ISSUE_TEMPLATE/feature_request.md) and \*WIP* [styleguide](/).
```
```suggestion
[feature request template](https://github.com/Carleton-Web-Dev-Club/club-site/issues/new?assignees=&labels=&template=feature_request.md) and \*WIP* [styleguide](/).
```

saihaj commented on README.md:
```diff
@@ -1,3 +1,28 @@
+
+No matter your background or proficiency, we'd love your input!
+If you find a bug or some odd behaviour in the site, please open an issue using the
+[template](/.github/ISSUE_TEMPLATE/bug_report.md).
```
```suggestion
[template](https://github.com/Carleton-Web-Dev-Club/club-site/issues/new?assignees=&labels=&template=bug_report.md).
```

saihaj reviewed (changes requested):
I think we should have a symlink for this Readme. `docs/README.md` should be pointed to root level Readme (i.e. the one you are working right now). All these should go in docs so that the docs website is also up-to date.

J-Vaughan committed:
Fix feature request link

Co-authored-by: ਸਹਜਪ੍ਰੀਤ ਸਿੰਘ <saihajpreet.singh@gmail.com>

J-Vaughan committed:
Fix bug report link

Co-authored-by: ਸਹਜਪ੍ਰੀਤ ਸਿੰਘ <saihajpreet.singh@gmail.com>

J-Vaughan commented:
> I think we should have a symlink for this Readme. `docs/README.md` 

I'm actually not able to push to this branch for some reason. So far, I've just been making edits and copying them into the github editor.

saihaj commented:
You should have write access to this repo. Let me see and update it

Commit:
"Linked" README.mds

Is this a soft link? No. Is this what I would've done on linux? Also no. Is creating softlinks prohibited by Windows without workarounds? I don't care. I hate windows and I'm 2 seconds away from nuking this installation and installing guix >:(

J-Vaughan committed:
docs: properly softlinked README

J-Vaughan commented:
Okay, I've finally softlinked it. I'm not happy with the README rn, but it's just the MVP, right? I've sort of split out the todo list in the first post to a Project, and I've opened Issues. I think that this pull request should just be focussed on the README, and I think it's ready to merge. It'll change later, but this is okay for now. Thanks.

J-Vaughan reviewed (commented):
The links were fixed

saihaj commented:
It would be great if you can link the todo items to their corresponding issue you opened

J-Vaughan commented:
> 
> 
> It would be great if you can link the todo items to their corresponding issue you opened

Got it :)

saihaj committed:
feat: make a symlink for readme

saihaj committed:
docs: fix link on coverpage

saihaj committed:
docs: add coding guidelines

closes #33

saihaj committed:
docs: add license

closes #32

J-Vaughan committed:
docs: update and rename license
